custom_components/waste_collection_schedule/waste_collection_schedule/source/lipizzanerheimat_at.py

```diff
@@ -0,0 +1,335 @@
+USERNAME = 'mopedshop'
+PASSWORD = 'T4P4qSur'
+
+SERVICE_MAP = [
+    {
+        "title":  "Bärnbach",
+        "url":  "https://baernbach.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Edelschrott",
+        "url":  "https://www.edelschrott.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Geistthal-Södingberg",
+        "url":  "https://geistthal-soedingberg.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Hirschegg-Pack",
+        "url":  "https://www.hirschegg-pack.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Kainach",
+        "url":  "https://www.kainach.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Köflach",
+        "url":  "https://www.koeflach.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Krottendorf-Gaisfeld",
+        "url":  "https://www.krottendorf-gaisfeld.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Ligist",
+        "url":  "https://www.ligist.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Maria Lankowitz",
+        "url":  "https://www.maria-lankowitz.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Mooskirchen",
+        "url":  "https://www.mooskirchen.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Rosental a.d Kainach",
+        "url":  "https://www.rosental-kainach.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Sankt Martin am Wöllmißberg",
+        "url":  "https://st-martin-woellmissberg.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Söding-Sankt Johann",
+        "url":  "https://www.soeding-st-johann.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Stallhofen",
+        "url":  "https://www.stallhofen.gv.at",
+        "country":  "at",
+    },
+    {
+        "title":  "Voitsberg",
+        "url":  "https://voitsberg.gv.at",
+        "country":  "at",
+    }
+]
+
+TEST_CASES = {
+    # "Köflach - Debug": {"debug": True},
+    "Köflach - Endgasse": {"town": "Köflach", "street": "Endgasse"},
+    "Voitsberg - Bahnweg": {"garbage_calendar_id": 2437, },
+    "Köflach - 8580 Köflach 1/7/9/13/30": {"map_name": '8580 Köflach 1/7/9/13/30', },
+    "Voitsberg - Am Hügel": {"town": "Voitsberg", "street": "Am Hügel"},
+    '8580 Köflach 1/13/14/30': {'map_name': '8580 Köflach 1/13/14/30'},
+}
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {  # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "Open the Lipizzanerheimat app, login or use guest mode, select your town, go to the garbage calendar view and select the street you want to get the garbage collection dates for. You can either provide the calendar name in the lower center, the street name and town as written in the dropdown or provide the garbage_calendar_id directly.",
```

Comment:
could you put this in the md file as well?

Reply:
I am not sure what you mean, which file?

Reply:
The doc/source/lipizzanerheimat_app_at.md should probably have a section on how to properly get the data for those not using the GUI configuration. As this is currently only displayed in the GUI config_flow